### ISSUE TITLE
Mutation test report showing zero mutation score

### ISSUE THREAD

**Issue by LokeshGangaiah** [stryker-mutator/stryker-js#2686]
**Question**

Why would my report show the score has zero ? please see the attached screen shot

![Screen Shot 2021-01-06 at 10 49 26 am](https://user-images.githubusercontent.com/12905101/103712311-920c2b00-500d-11eb-8b56-cbfb3c7b1dd6.png)

**Stryker environment**

```
npm ERR! peer dep missing: @angular/common@^8.0.0, required by @ng-idle/core@8.0.0-beta.4
npm ERR! peer dep missing: @angular/core@^8.0.0, required by @ng-idle/core@8.0.0-beta.4
npm ERR! peer dep missing: @ngrx/store@8.6.1, required by @ngrx/store-devtools@8.6.1
npm ERR! peer dep missing: node-sass@^4.0.0, required by sass-loader@8.0.2
npm ERR! peer dep missing: fibers@>= 3.1.0, required by sass-loader@8.0.2
npm ERR! peer dep missing: html-webpack-plugin@^2.21.0 || ~3 || >=4.0.0-alpha.2 <5, required by webpack-subresource-integrity@1.4.0
npm ERR! peer dep missing: bufferutil@^4.0.1, required by ws@7.4.2
npm ERR! peer dep missing: utf-8-validate@^5.0.2, required by ws@7.4.2
npm ERR! peer dep missing: bufferutil@^4.0.1, required by ws@7.4.2
npm ERR! peer dep missing: utf-8-validate@^5.0.2, required by ws@7.4.2
npm ERR! peer dep missing: bufferutil@^4.0.1, required by ws@7.4.2
npm ERR! peer dep missing: utf-8-validate@^5.0.2, required by ws@7.4.2
├─┬ @stryker-mutator/core@4.3.1
│ ├─┬ @stryker-mutator/api@4.3.1
│ ├─┬ @stryker-mutator/instrumenter@4.3.1
│ │ ├── @stryker-mutator/api@4.3.1 deduped
│ │ ├── @stryker-mutator/util@4.3.1 deduped
│ ├─┬ @stryker-mutator/util@4.3.1
├─┬ @stryker-mutator/karma-runner@4.3.1
│ ├── @stryker-mutator/api@4.3.1 deduped
│ ├── @stryker-mutator/util@4.3.1 deduped
├─┬ @stryker-mutator/typescript-checker@4.3.1
│ ├── @stryker-mutator/api@4.3.1 deduped
│ ├── @stryker-mutator/util@4.3.1 deduped
```

```
  "karma": "~5.1.0",
```

**Additional context**

this will not happen if i run or include more test scenarios or something like that i am not sure need to debug why

**Comment by simondel** [comment]
All the mutants have the state _survived_ this means no test failed when they were put into your code. Because of this, you get a score of 0%. If you add (more) tests for the code that is being mutated, your score might increase because tests may fail when the mutant is inserted.

More info can be found here: https://stryker-mutator.io/docs/

**Comment by nicojs** [comment]
Are you using [app-root-path](https://www.npmjs.com/package/app-root-path) or [NODE_PATH](https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders) to redefine imports? If so, Stryker might be running your un-mutated code, instead of running your code in the sandbox.

**Comment by LokeshGangaiah** [comment]
> Are you using [app-root-path](https://www.npmjs.com/package/app-root-path) or [NODE_PATH](https://nodejs.org/api/modules.html#modules_loading_from_the_global_folders) to redefine imports? If so, Stryker might be running your un-mutated code, instead of running your code in the sandbox.


@nicojs not using app-root-path/node_path, the stryker is running in the sandbox, see the logs from the run:
```

10:28:45 (70977) INFO ConcurrencyTokenProvider Creating 1 test runner process(es).
10:28:45 (70977) DEBUG Sandbox Creating a sandbox for files in /Users/lokesh/git/front-end-api/.stryker-tmp/sandbox8768767
10:28:45 (70977) INFO DryRunExecutor Starting initial test run. This may take a while.
```

**Comment by nicojs** [comment]
I think I've found the cause while I was implementing `--inPlace` mode and fixed it, see #2706

@LokeshGangaiah please try again with @stryker-mutator/karma-runner@4.40
